servercfg/config.go

```diff
@@ -35,7 +40,55 @@ const (
 	OverrideDataDirKey = "data_dir"
 )
 
-var ConfigHelp = `Place holder. This will be replaced with details on config.yaml options`
+// LogLevel defines the available levels of logging for the server.
+type LogLevel string
+
+const (
+	LogLevel_Trace   LogLevel = "trace"
+	LogLevel_Debug   LogLevel = "debug"
+	LogLevel_Info    LogLevel = "info"
+	LogLevel_Warning LogLevel = "warning"
+	LogLevel_Error   LogLevel = "error"
+	LogLevel_Fatal   LogLevel = "fatal"
+)
+
+const (
+	DefaultHost                    = "localhost"
+	DefaultPort                    = 5432
+	DefaultUser                    = ""
+	DefaultPass                    = ""
+	DefaultTimeout                 = 8 * 60 * 60 * 1000 // 8 hours, same as MySQL
```

Comment:
Saying our timeout is the same as MySQL's just _feels_ wrong but seems okay lol

Reply:
I'm gonna fix that next, I don't think the doltgres listener even uses this